level_2/section_4.md

```diff
@@ -1,84 +1,30 @@
-# Section 4: More Advanced Survival Techniques
-
-## Enchanting
+# Section 4: Enchanting
 
 Enchanting is a mechanic that augments armor, tools, weapons and books with one or more of a variety of enchantments that improve an item's existing abilities or imbue them with additional abilities and uses.
 
-Enchanting is a very important technique in Minecraft. Enchanted tools, armor and weapons make success in your Minecraft adventures much more likely, and are more fun to use as well.
+Enchanting is a very important technique in Minecraft. Enchanted tools, armor and weapons make success in your Minecraft adventures much more likely, and makes items more fun to use as well.
+
+In order to enchant anything, experience is required. The easiest way to gain experience is by mining items such as coal and redstone, or raising animals. The most experience is gained, however, by fighting mobs.
 
 To enchant an item, right-click on an enchanting table and place the item and 1–3 lapis lazuli in the input slots. Options will display on the right-hand side of the enchanting display.
```

Comment:
Should we provide the crafting recipe for an enchanting table (and a bookshelf, for that matter)?

Reply:
(same question for rails, minecarts, etc.)